ComponentTextKitApplicationTests/CKTextComponentTests.mm

```diff
@@ -407,4 +426,31 @@ - (void)testTruncationStringLargerThanLastLine
   CKSnapshotVerifyComponent(c, kUnrestrictedSize, @"");
 }
 
+- (void)testShouldUseCustomLayoutManagerClass
+{
+	NSString *contentString = @"This is a string with a border behind every other word";
+  NSMutableAttributedString *attributed = [[NSMutableAttributedString alloc] initWithString:contentString attributes:@{NSFontAttributeName : [UIFont fontWithName:@"HelveticaNeue" size:12.5]}];
+  __block NSUInteger index = 0;
+  [contentString enumerateSubstringsInRange:[contentString rangeOfString:contentString]
+                                    options:NSStringEnumerationByWords
+                                 usingBlock:^(NSString *substring, NSRange substringRange, NSRange enclosingRange, BOOL *stop) {
+                                   if (index++ % 2) {
+                                     [attributed addAttribute:NSBackgroundColorAttributeName value:[UIColor greenColor] range:substringRange];
```

Comment:
The verification here is that I specify `+[UIColor redColor]` as the background color attribute, which I then override with green in the layout manager subclass. Ergo, the subclass is used.